src/Service/StaleIssueCommentGenerator.php

```diff
@@ -71,6 +71,9 @@ private function feature(): string
         return <<<TXT
 Thank you for this suggestion.
 There has not been a lot of activity here for a while. Would you still like to see this feature?
+[Every feature request may or may not be developed in future versions based on the community.](https://symfony.com/doc/current/contributing/community/releases.html#development)
```

Comment:
I see you took that sentence from the website. But it is a strange sentence.. Maybe we can make it simpler?

```suggestion
[Every feature is developed by the community.](https://symfony.com/doc/current/contributing/community/releases.html#development)
```

Reply:
Agreed